docs/tips-and-tricks.rst

```diff
@@ -41,6 +41,16 @@ Then you deploy the commit::
    --commit=ec07ad6c54e803d1428e5580426a41315e50a14376af033458e7a65bfb2b64f0 \
    org.gnome.Recipes
 
+If the application is installed per-user, you will get the error::
+
+ error: org.gnome.Recipes not installed
+
+In that case, omit `sudo` from the command::
+
+ $ flatpak update \
+   --commit=ec07ad6c54e803d1428e5580426a41315e50a14376af033458e7a65bfb2b64f0 \
+   org.gnome.Recipes
+
```

Comment:
This is a bit verbose and unclear. I think I'd go with something like:

```suggestion
.. note::
   The example here uses ``sudo`` for system installations because, unlike normal upgrades, downgrades are considered a privileged action. If the application is installed per-user you would run it as that user.
```